maintaining/breaking-changes-guide.md

```diff
@@ -35,6 +35,8 @@ We recommend that:
   `incompatible_`).
 
 * You add the label [`incompatible-change`](https://github.com/bazelbuild/bazel/labels/incompatible-change).
+  This ensures that the flag, once it's merged, will be picked up by the
+  [`bazelisk-plus-incompatible-flags` pipeline](https://buildkite.com/bazel/bazelisk-plus-incompatible-flags).
```

Comment:
Are we going to keep this pipeline?
It is not that useful, so it's not clear if mentioning it helps people.

Reply:
FWIW, I'm using that pipeline as a signal to check whether a Build language change (immutable existing rules view) breaks any Bazel rule sets.